subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/IntegrationTestBuildContext.java

```diff
@@ -72,6 +72,10 @@ public GradleVersion getVersion() {
         return GradleVersion.current();
```

Comment:
Isn't this not quite right because we're relying on the test version info being on the classpath? Couldn't we make this always use `integTest.distZipVersion`?